### PR TITLE
created the basic project structure, automated build, installation and testing, fixes #1294

### LINKED ISSUE #1294: Basic plumbing for revamping Python integration
create some basic project/code structure for the more pythonic Deephaven Python package

- a separate project folder/structure, avoid disrupting the current implementation and the continuous development in it
- basic build script, maybe some Gradle integration
- a README that clearly states the project is a WIP and for internal use only for the foreseeable future

Part of #1263

### PR DESCRIPTION
**Pull request by jmao-denver**
1. created a project 'pyintegration' structured as a typical Python project, and completely separate from the current server-side Python project so that both can be used and evolve at the same time, until the switch off is ready
2. created setup.py and README for the new project
3. created a tests folder for unit/integration tests
4. changed Integrations/build.gradle to built, install, test the new package in Docker

### CODE REVIEW

**Comment by jmao-denver** [comment]
I have read the CLA Document and I hereby sign the CLA

File: Integrations/build.gradle [hunk 1]
```diff
@@ -63,16 +63,30 @@ idea {
...
-task prepareDockerForWheel(type: Sync) {
-    from 'python'
-    into layout.buildDirectory.dir('docker')
-}
-def buildDockerForWheel = Docker.registerDockerImage(project, 'buildDockerForWheel') {
-    dependsOn prepareDockerForWheel
-    buildArgs.put('DEEPHAVEN_VERSION', "${project.version}")
-    images.add('deephaven/deephaven-wheel:local-build')
-    target.set('build')
+def buildPyWheel = { String task_name, String image_name, String source_path ->
```
Comment (niloc132) [inline Integrations/build.gradle:66]: java nit: `taskName`, `imageName`, `sourcePath`

Comment (jmao-denver) [reply Integrations/build.gradle:66]: done

File: Integrations/build.gradle [hunk 2]
```diff
@@ -63,16 +63,30 @@ idea {
...
+    Docker.registerDockerTask(project, task_name) {
+        copyIn {
+            from(source_path) {
+                into '/src'
```
[the remark this answers is not in the document]
Comment (jmao-denver) [reply Integrations/build.gradle:70]: done

**Review by niloc132 — APPROVED** [review]

**Pull request merged**